packages/paste-website/src/pages/components/date-picker/index.mdx

```diff
@@ -0,0 +1,617 @@
+---
+title: Date Picker - Components
+description: A form component used to select a date.
+slug: /components/date-picker/
+---
+
+import {graphql} from 'gatsby';
+import Changelog from '@twilio-paste/date-picker/CHANGELOG.md';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Label} from '@twilio-paste/label';
+import {HelpText} from '@twilio-paste/help-text';
+import {DatePicker, formatReturnDate} from '@twilio-paste/date-picker';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Stack} from '@twilio-paste/stack';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {useUID} from '@twilio-paste/uid-library';
+import {Combobox} from '@twilio-paste/combobox';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {
+  RequiredDatePicker,
+  DateRangePicker,
+  OnChangePicker,
+  HasErrorPicker,
+  DisabledPicker,
+  ReadOnlyPicker,
+  LabelOnlyPicker,
+  StackOfPickers,
+  MinAndMaxPicker,
+} from '../../../component-examples/DatePickerExamples.ts';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/date-picker"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/date-picker/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Date Picker"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/date-picker"
+  storybookUrl="/?path=/story/components-date-picker--default-date-picker"
+  data={props.data.allPasteComponent.edges}
```

Comment:
Also add `packageStatus={props.data.allAirtable.edges}` after this line.

Reply:
Thanks! I'll make these changes after rebasing